firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionCoordinator.kt

```diff
@@ -47,6 +48,12 @@ internal class SessionCoordinator(
         }
 
       Log.i(TAG, "Initiate session start: $sessionEvent")
```

Comment:
Can you move this log to after the `.log` in the try below? Then we either get a log of success or failure on the session start being sent.

Reply:
Done